core/async/src/test_loop/sender.rs

```diff
@@ -101,7 +101,7 @@ where
 impl<M, R, A> CanSend<MessageWithCallback<M, R>> for TestLoopSender<A>
 where
     M: actix::Message<Result = R> + Debug + Send + 'static,
```

Comment:
do we still need this M bound?

Reply:
I tried to trivially remove it and it giving some errors. Let us come back to this when we are getting rid of `actix::Message` in it's entirety.